txtorcon/torconfig.py

```diff
@@ -423,6 +424,11 @@ class EphemeralHiddenService(object):
     https://gitweb.torproject.org/torspec.git/tree/control-spec.txt#n1295
     '''
 
+    @classmethod
+    def _is_valid_keyblob(cls, key_blob_or_type):
+        return (isinstance(key_blob_or_type, str) and
+                re.match(r'[^ :]+:[^ :]+$', key_blob_or_type))
```

Comment:
Should this be `six.text_type`? or should it be `bytes`?

Currently, the only existing thing happens to be ASCII, leading me to believe that `text_type` is more-correct. Also, it'll typically be "stuff a user types in" also leading to `text_type`...that'll mean it demands `unicode` on Py2.